modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -383,7 +379,7 @@ export default class Combobox extends React.Component<ComboboxProps, ComboboxSta
           {showClearButton && (
             <ResetButton
               shouldShow={!!this.state.value}
-              aria-label={this.props.clearButtonLabel}
+              aria-label={clearButtonLabel}
```

Comment:
maybe a bit more descriptive name, clearButtonLabel might mean many things, but it is actually aria-label, so maybe clearButtonAriaLabel